tests/unit/proxy-api/proxy-api.service.spec.ts

```diff
@@ -10,14 +10,15 @@ import configuration from '../../../src/config/configuration.test';
 jest.mock('../../../src/confluence/confluence.service');
 jest.mock('../../../src/jira/jira.service');
 
+// eslint-disable-next-line @typescript-eslint/ban-types
+type DeepPartial<T> = T extends object
```

Comment:
I'm not 100% sure about what is going on there. Could you explain this part a little bit?